pkg/releaseutil/manifest.go

```diff
@@ -42,8 +42,13 @@ func SplitManifests(bigfile string) map[string]string {
 	res := map[string]string{}
 	// Making sure that any extra whitespace in YAML stream doesn't interfere in splitting documents correctly.
 	docs := sep.Split(bigfile, -1)
-	for i, d := range docs {
-		res[fmt.Sprintf(tpl, i)] = d
+	var count int = 0
+	for _, d := range docs {
```

Comment:
Do you think we need to do `strings.TrimSpace(d)`? Or is there no risk of getting a document that's just whitespace?